### DISCUSSION TITLE
大家都是在mac 上进行调试的吗？有没有在windows 上进行调试的

### DISCUSSION THREAD

**Discussion by hujunhui530** [apache/streampark#2226] [Q&A] [answered]
在windows 上调试出现路径URI 异常，因为不是/opt/xxx 这种 ，而是 D:\A\B\C 这种，有没有什么解决方案，还有就是 yarn application 模式下，生成的环境变量的连接串 跟linux 上也不一致，这样提交到远程的linux 上的yarn 时，也会运行失败，有没有解决方案？

**Comment by NightRunner** [comment]
Me too. I install ubuntu 22.04 LTS on my thinkpad to solute this problem. Streampark not support windows OS.

**Comment by caicancai** [comment, marked as answer]
streampark does not currently support windows. Of course, you can use wsl or change the source code of streampark to try it out, it may be a good idea
<img width="463" alt="屏幕截图 2023-08-23 122643" src="https://github.com/apache/incubator-streampark/assets/77189278/2397644c-ce47-4fc1-8feb-d9a541dc109b">

**Comment by VampireAchao** [comment]
I have already bought a Mac for this project 🤑

**Comment by wuyu8995861** [comment]
我在WSL上安装的Flink，报错Flink_HOME无效，是咋回事呢

**Comment by Cqz666** [comment]
真是坑，服务部署wsl，window访问，setting页面死活都是404

**Reply by caicancai** [reply]
It is recommended to use linux and mac for deployment, windows we have said on the official website that the support is not very perfect

**Comment by linweijiang** [comment]
Highly recommend adapting to Windows; otherwise, it will deter a significant number of developers, which is not conducive to community development.

**Reply by caicancai** [reply]
The community has been following up on this matter and striving to complete it as soon as possible
